app/controllers/users.js

```diff
@@ -41,3 +42,24 @@ exports.getUsers = async (req, res, next) => {
     next(error);
   }
 };
+exports.adminLog = async (req, res, next) => {
+  try {
+    const userObject = req.body;
+    const userFound = await userService.findUser(userObject);
+    if (userFound === null) {
+      const password = helper.passEncryptred(userObject.password);
+      const userCreated = varParse.nameParse(userObject, password);
+      const admin = await helper.adminRole(userCreated);
+      await userService.createUser(admin);
+      logger.info('Admin created: ', admin.firstName);
+      res.status(201).send(admin);
```

Comment:
En el método nameParse() pusimos la variable `admin` como argumento, para que cuando le enviemos ese argumento en `true`, ya organice el código para ser almacenado.. Habiendo dicho esto, entonces lo podemos dejar como: 

```suggestion
      const userToCreate = varParse.nameParse(userObject, password, true);
      await userService.createUser(userToCreate);
      logger.info('Admin created: ', userToCreate.firstName);
      res.status(201).send(userToCreate);
```